source/index.html.md

```diff
@@ -65,6 +65,8 @@ includes:
   - fpd
   - capture_external_event_fpd
   - capture_external_event_chat
+  - etl_reports
+   - etl_reports/etl_lead_attribution
```

Comment:
This isn't right, it results in in an error
```
Error: Could not locate partial: includes/etl_reports - etl_reports/_etl_lead_attribution
```
You have a space character before the -, remove it and it works